### PR TITLE
Fix Headless deployment

### PR DESCRIPTION
**Pull request by vesko-tyk**

### CODE REVIEW

File: tyk-headless/templates/deployment-pmp.yaml [hunk 1]
```diff
@@ -47,14 +48,22 @@ spec:
         env:
           - name: REDIGOCLUSTER_SHARDCOUNT
             value: "{{ .Values.redis.shardCount }}"
+            {{- if .Values.mongo.enabled }}
```
Comment (joshblakeley) [inline tyk-headless/templates/deployment-pmp.yaml:51]: Wrap all of these in a single if statement since they all follow the same behaviour

**Review by joshblakeley — CHANGES_REQUESTED** [review]
Once small comment but otherwise great thanks @vesko-tyk !

File: tyk-headless/templates/deployment-pmp.yaml (hunk 1 above)
Comment (vesko-tyk) [reply tyk-headless/templates/deployment-pmp.yaml:51]: done

**Review by joshblakeley — APPROVED** [review]
💯

**Pull request merged**